pkg/api/apis/operators/v1alpha1/clusterserviceversion_test.go

```diff
@@ -282,3 +284,111 @@ func TestSupports(t *testing.T) {
 		})
 	}
 }
+
+func TestSetPhaseWithConditionsLengthAtLimit(t *testing.T) {
```

Comment:
These tests look good, but we try to keep a consistent style for tests. Generally we do a "table-drive" test style, where each test case defines expected inputs/outputs.

So for these tests, that would look something like 

```go
func TestSetPhaseLengthLimits() {
  tests := []struct{
     name string
     currentConditions []conditions
     args args
     expected expected
  } {
    // an instance for each test case
  }
for _, tt := range tests {
  t.Run(tt.name, func(t *testing.T) {
    // set csv conditions to tt.currentConditions
    // call csv.SetPhase(args)
    // check csv conditions == expected
  }
}
```

Reply:
@ecordell  converted to table driven, it looks concise and more readable now. Can you take a look?